app/scripts/views/sign_up.js

```diff
@@ -199,6 +212,10 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
         });
     },
 
+    passwordCheck: function () {
+      passwordcheck(this, this.$el.find('#password'), this.metrics, this.translator, bloom);
```

Comment:
access to View's global 'bloom' , related to https://github.com/mozilla/fxa-content-server/pull/2810/files#r35327891

should we make `this.blookFilter`, also `bloom` can be `null`, do we want to  call `passwordcheck` then?